extensions/wikia/MercuryApi/handlers/MercuryApiCategoryHandler.class.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+class MercuryApiCategoryHandler {
+
+	public static function getCategoryContent(Title $title) {
+		$categoryPage = CategoryPage::newFromTitle($title, RequestContext::getMain());
+		return [
+			'members' => self::getMembers($categoryPage),
+			'exhibition' => self::getExhibition($categoryPage),
```

Comment:
don't set exhibition - it's not handled on front-end - no need to send this data